app/controllers/spree/admin/return_index_controller.rb

```diff
@@ -13,14 +13,14 @@ def customer_returns
 
       private
 
-      def collection(resource)
+      def collection(resource, store_id_query_key = nil)
         return @collection if @collection.present?
 
         params[:q] ||= {}
+        params[:q][store_id_query_key] ||= current_store.id if store_id_query_key
```

Comment:
@aleksandarpetrushev please use `for_strore` scope and not ransack parameters which can be changed via URL